src/useCroods.tsx

```diff
@@ -103,7 +108,7 @@ const useCroods = ({
   )
 
   const destroy = useCallback(
-    contextOpts => async (query?: object) => {
+    contextOpts => async (query: object = {}, requestConfig: object = {}) => {
```

Comment:
I'm not sure `object` is permissive enough in Typescript.. I had problems with it before.
I ended up using either `any` which is bad or some version of:
```js
interface MyObj {
  someAttrIKnow?: string
  [key: string]: any
}
```